### ISSUE TITLE
Support for commonjs exports.xxx?

### ISSUE THREAD

**Issue by wclr** [developit/workerize-loader#20]
Seems commonjs modules are not supported? 

```js
module.exports.xxx = function () {
   return 1
}
```
or 
```js
exports.xxx = () => {
   ...
}

```

Is there a reason for this?

**Comment by developit** [comment]
Yes - workerize only recognizes ES Modules.  I'm open to changing that, making the export detection work at runtime instead. It will require making module instantiation asynchronous though.
